packages/addons/service/nextpvr/source/bin/nextpvr-downloader

```diff
@@ -61,6 +61,9 @@ fi
 # extract NextPVR
 kodi-send --action="Notification(Extracting NextPVR,starting,1000,${ICON})" >/dev/null
 mkdir -p ${ADDON_DIR}/nextpvr-bin
+
+ln -s ${ADDON_DIR}/bin/libmediainfo.so ${ADDON_DIR}/nextpvr-bin
```

Comment:
I'm not in favour of this - but if that's how nextpvr works - and if grabs the .so file from nextpvr-bin - then ok. But should reference the lib.private directory. hdhomerun_config should be done similarly - it could be in bin.private and linked into nextpvr-bin. Both the .so file and the hdhomerun_config should not be added to the LE PATH.

Reply:
It is dotnet doing a specific library search (it even checks the dotnet library folder) but the executable search don't work the same way.  I will check with the author if the default folder set be LE https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/addons/service/nextpvr/source/bin/nextpvr.start#L25 means it will be reliably found.

Reply:
Actually it looks like the current working directory will be set ok  so I can probably use the symbolic link from bin.private so I will set that up tomorrow.  Thanks.

```

[code]
2024-09-26 23:44:25.913	[ERROR][26]	Unexpected error doing legacy HDHR tuning: System.ComponentModel.Win32Exception (2): An error occurred trying to start process 'hdhomerun_config' with working directory '/storage/.kodi/addons/service.nextpvr/nextpvr-bin'. No such file or directory
[/code]
```